buildenvs/base.Dockerfile

```diff
@@ -0,0 +1,143 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2022, NEC Europe Ltd., Unikraft GmbH, and The KraftKit Authors.
+# Licensed under the BSD-3-Clause License (the "License").
+# You may not use this file except in compliance with the License.
+
+ARG REG=index.unikraft.io
+
+ARG UK_ARCH=x86_64
+
+ARG GCC_VERSION=12.2.0
+ARG GCC_SUFFIX=
+
+ARG QEMU_VERSION=7.1.0
+
+ARG KRAFTKIT_VERSION=latest
+
+ARG DEBIAN_VERSION=bullseye-20221114
+
+FROM ${REG}/gcc:${GCC_VERSION}-x86_64${GCC_SUFFIX} AS gcc-x86_64
+# FROM ${REG}/gcc:${GCC_VERSION}-arm${GCC_SUFFIX}    AS gcc-arm
+# FROM ${REG}/gcc:${GCC_VERSION}-arm64${GCC_SUFFIX}  AS gcc-arm64
+
+FROM ${REG}/qemu:${QEMU_VERSION}                   AS qemu
+FROM ${REG}/myself:${KRAFTKIT_VERSION}             AS kraft
+
+# Copy the kraft tool
+COPY . /go/src/kraftkit.sh
+
+# Build the kraft tool - also installs it to the environment
+WORKDIR /go/src/kraftkit.sh
+
+RUN set -xe; \
+  make kraft;
```

Comment:
Since we're `COPY`ing `--from` `kraft` we don't need to run these steps:
```diff
- # Copy the kraft tool
- COPY . /go/src/kraftkit.sh
- 
- # Build the kraft tool - also installs it to the environment
- WORKDIR /go/src/kraftkit.sh
- 
- RUN set -xe; \
-   make kraft;
```